<?xml version="1.0" encoding="UTF-8"?><testsuite name="Firefox_340_Mac_OS.localStorageWrapper" errors="1" failures="0" tests="1" time="0.0">
<testcase classname="Firefox_340_Mac_OS.localStorageWrapper" name="test test for wrapper function" time="0.0">
<error type="error">[{"message":"assertion outside test context, was @http://localhost:9876/test/test/qunit/model/database/localStorageWrapper_test.js:16:24\n@http://localhost:9876/test/test/qunit/QUnitAdapter.js:47:28\njstestdriver.plugins.TestRunnerPlugin.prototype.runTest@http://localhost:9876/static/runner.js:5403:7\nrunTest@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:201:21\ncoverage&lt;/TestResultIterator.prototype.runNext@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:292:5\ncoverage&lt;/InstrumentedTestCaseRunner.prototype.run@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:250:7\ncoverage&lt;/InstrumentedTestCaseRunnerPlugin.prototype.runTestConfiguration@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:221:5\njstestdriver.PluginRegistrar.prototype.dispatch_@http://localhost:9876/static/runner.js:1603:11\njstestdriver.PluginRegistrar.prototype.runTestConfiguration@http://localhost:9876/static/runner.js:1693:3\njstestdriver.TestRunner.prototype.runConfiguration@http://localhost:9876/static/runner.js:2817:1\njstestdriver.TestRunner.prototype.runNextConfiguration_@http://localhost:9876/static/runner.js:2795:1\nbound@http://localhost:9876/static/jstestdrivernamespace.js:71:12\ncoverage&lt;/InstrumentedTestCaseRunner.prototype.run@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:248:7\nbound@http://localhost:9876/static/jstestdrivernamespace.js:71:12\n","name":"Error","fileName":"http://code.jquery.com/qunit/qunit-1.12.0.js","lineNumber":917,"stack":".push@http://code.jquery.com/qunit/qunit-1.12.0.js:917:1\nassert.equal@http://code.jquery.com/qunit/qunit-1.12.0.js:556:3\n@http://localhost:9876/test/test/qunit/model/database/localStorageWrapper_test.js:16:24\n@http://localhost:9876/test/test/qunit/QUnitAdapter.js:47:28\njstestdriver.plugins.TestRunnerPlugin.prototype.runTest@http://localhost:9876/static/runner.js:5403:7\nrunTest@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:201:21\ncoverage&lt;/TestResultIterator.prototype.runNext@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:292:5\ncoverage&lt;/InstrumentedTestCaseRunner.prototype.run@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:250:7\ncoverage&lt;/InstrumentedTestCaseRunnerPlugin.prototype.runTestConfiguration@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:221:5\njstestdriver.PluginRegistrar.prototype.dispatch_@http://localhost:9876/static/runner.js:1603:11\njstestdriver.PluginRegistrar.prototype.runTestConfiguration@http://localhost:9876/static/runner.js:1693:3\njstestdriver.TestRunner.prototype.runConfiguration@http://localhost:9876/static/runner.js:2817:1\njstestdriver.TestRunner.prototype.runNextConfiguration_@http://localhost:9876/static/runner.js:2795:1\nbound@http://localhost:9876/static/jstestdrivernamespace.js:71:12\ncoverage&lt;/InstrumentedTestCaseRunner.prototype.run@http://localhost:9876/test/com/google/jstestdriver/coverage/javascript/LCOV.js:248:7\nbound@http://localhost:9876/static/jstestdrivernamespace.js:71:12\n"}]</error>
</testcase>
</testsuite>
